src/content/documentation/publish/create-an-appealing-listing.md

```diff
@@ -293,6 +293,86 @@ And, when users suggest improvements or highlight issues in the reviews, use the
 
 <!-- Single Column Body Module -->
 
+<section id="make-use-of-markdown" class="module">
+<article class="module-content grid-x grid-padding-x">
+<div class="cell small-12">
+
+## Make use of Markdown
+
+Some fields in your add-on's listing support a limited set of Markdown.
```

Comment:
Can we be any more specific amount which fields these are? Otherwise this leave the reader to experiment and discover for themselves.